setup.py

```diff
@@ -79,6 +79,7 @@
         'bitcoin==1.1.42',
         'flask==0.10.1',
         'requests==2.9',
+        'gunicorn',
```

Comment:
OK, I won my laziness and actually we can use the [`~` operator](https://www.python.org/dev/peps/pep-0440/#compatible-release). It's totally rad. We can do "gunicorn~=19.0" and it'd install the last version of `gunicorn` with major version `19`.

I also [filed an issue](https://github.com/benoitc/gunicorn/issues/1240) to ask if they use semantic versioning, so we can safely assume that if we pin version `19` we won't encounter compatibility issues.

For now I'll pin version `19` tho.